wallet/src/wallet/mod.rs

```diff
@@ -220,6 +221,8 @@ pub enum WalletError {
     FailedToConvertPartiallySignedTx(PartiallySignedTransaction),
     #[error("The specified address is not found in this wallet")]
     AddressNotFound,
+    #[error("The specified standalone address is not found in this wallet")]
+    StandaloneAddressNotFound,
```

Comment:
Why not include the address? Most of these errors can take more information in them.